test/conftest.py

```diff
@@ -24,7 +29,7 @@ def load_ee_plugin(qgis_app, setup_ee):
     QSettings().setValue("locale/userLocale", "en")
 
     # Initialize and register the plugin
-    plugin = GoogleEarthEnginePlugin(qgis_app)
+    plugin = ee_plugin.GoogleEarthEnginePlugin(qgis_app, ee_config=ee_config)
```

Comment:
Love it. Sets us up nicely for testing different scenarios like switching between projects.